easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -45,6 +46,21 @@
 public class SCMStepdefs {
 
     protected static final EasyCukesConfiguration<CommonConfigurationBean> configuration = new EasyCukesConfiguration<>(CommonConfigurationBean.class);
+    
+	/*
+	 *  Used to convert base url into scm base url 
+	 */
+	private static final String SCM_BASE_URL = "/portal";
+	
+	/*
+	 *  Used to log message after commit
+	 */
+	private static final String COMMIT_DONE_MESSAGE = "Done Commit...";

```

Comment:
All the other log messages are directly written within the loggers. No need to put the log messages in constants.